pontoon/batch/actions.py

```diff
@@ -0,0 +1,200 @@
+from django.http import (
+    JsonResponse,
+)
+from django.utils import timezone
+
+from pontoon.base.models import (
+    Entity,
+    TranslationMemoryEntry,
+    Translation,
+)
+
+
+def get_translations_info(translations, locale):
+    """Return data about a translations set.
+
+    :arg QuerySet translations: a django QuerySet of Translation objects
+    :arg Locale locale: the Locale object of the current locale
+
+    :returns: a tuple with:
+              * the number of translations in the QuerySet
+              * a list of corresponding TranslatedResource objects
+              * a list of corresponding Entity objects
+
+    """
+    # Must be executed before translations set changes, which is why
+    # we need to force evaluate QuerySets by wrapping them inside list()
+    count = translations.count()
+    translated_resources = list(translations.translated_resources(locale))
+    changed_entities = list(
+        Entity.objects.filter(translation__in=translations).distinct()
+    )
+
+    return count, translated_resources, changed_entities
+
+
+def batch_action(form, user, translations, locale):
+    """Empty batch action, does nothing, only used for documentation.
```

Comment:
@adngdb what do you think about moving that part into function name I mean, to rename this function into:
`def empty_batch_action` or even `def default_action`  - that would be more explicit imho.